web_m2x_options/__openerp__.py

```diff
@@ -0,0 +1,125 @@
+# -*- coding: utf-8 -*-
+
+{
+    "name": 'web_m2x_options',
+    "version": "0.1",
+    "description": """
```

Comment:
You can remove this if you want (to avoid maintain the same text in two places), because now Odoo 8.0 recognizes rst files and parse them.